etna/articles/models.py

```diff
@@ -46,6 +46,9 @@ class ArticleIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
 
     new_label_end_date = datetime.now() - timedelta(days=21)
 
+    # DataLayerMixin overrides
+    gtm_content_group = "Stories from the archives"
+
```

Comment:
How about `gtm_content_group = HomePage.gtm_content_group`?

Reply:
Again, this is the value that was previously determined by [this code here]([set here](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/core/models/basepage.py#LL19-L23C2)). Why would we want this value to be different?

Reply:
As revised in the requirements,  `contentGroup1= "stories"` for all story pages i.e. Article index page, Article.